tooling/scripts/create-superapp.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.xml.*
+
+def rootPom = new XmlParser().parse(new File('pom.xml'))
+def camelQuarkusVersion = rootPom.version.text()
+def quarkusVersion = rootPom.properties.'quarkus.version'?.text()
+def mavenEnforcerPluginVersion = rootPom.properties.'maven-enforcer-plugin.version'?.text()
+
+// Get org.apache.camel.quarkus dependencies from camel-quarkus-bom
+def cqBomPath = "${System.properties['user.home']}/.m2/repository/org/apache/camel/quarkus/camel-quarkus-bom/${camelQuarkusVersion}/camel-quarkus-bom-${camelQuarkusVersion}.pom"
+def bom = new XmlParser().parse(new File(cqBomPath))
+def cqBomDependencies = bom.dependencyManagement.dependencies?.dependency?.findAll {
+    it.groupId.text().startsWith('org.apache.camel.quarkus') && !it.artifactId.text().contains('-support')
```

Comment:
In case this still selects too many artifacts, it might be worthwhile to try selecting all `org.apache.camel.quarkus:camel-quarkus-*-deployment` artifacts and then remove `-deployment` from artifactId. That should find only extensions.